packages/cli/test/integration/credentials.ee.test.ts

```diff
@@ -141,12 +138,18 @@ test('GET /credentials should return all creds for owner', async () => {
 	expect(Array.isArray(ownerCredential.sharedWith)).toBe(true);
 	expect(ownerCredential.sharedWith.length).toBe(3);
 
-	ownerCredential.sharedWith.forEach((sharee: IUser, idx: number) => {
+	// Fix order issue (MySQL might return items in any order)
+	const ownerCredentialsSharedWithOrdered = [...ownerCredential.sharedWith].sort(
```

Comment:
Everything derived from `response.body.data` ends up as `any`. We need to improve the test call maker to use `requests.d.ts` in future.